src/allDrugs.jl

```diff
@@ -251,3 +251,23 @@ function combin2drugs(
     end
     p
 end
+
+function blissCellNum(g1s, g2s; n=8)
+    num = zeros(n,4)
+    T=144
+    for i=1:4
+        num[:, i] = (g1s[T,1,i] .+ g2s[T,1,i]) .- (g1s[T,:,i] + g2s[T,:,i])
+    end
+    combined = zeros(n, n, 6)
+    for j = 1:n
+        # it goes column by column
+        combined[j,:,1] = -(num[:,1] .+ num[j,2] .- (num[:,1] .* num[j,2])) .+ (g1s[T,1,1] .+ g2s[T,1,1]) # lap w/ dox
+        combined[j,:,2] = -(num[:,1] .+ num[j,3] .- (num[:,1] .* num[j,3])) .+ (g1s[T,1,1] .+ g2s[T,1,1]) # lap w/ gem
+        combined[j,:,3] = -(num[:,1] .+ num[j,4] .- (num[:,1] .* num[j,4])) .+ (g1s[T,1,1] .+ g2s[T,1,1]) # lap w/ pac
+        combined[j,:,4] = -(num[:,2] .+ num[j,3] .- (num[:,2] .* num[j,3])) .+ (g1s[T,1,2] .+ g2s[T,1,2]) # dox w/ gem
+        combined[j,:,5] = -(num[:,2] .+ num[j,4] .- (num[:,2] .* num[j,4])) .+ (g1s[T,1,2] .+ g2s[T,1,2]) # dox w/ pac
+        combined[j,:,6] = -(num[:,3] .+ num[j,4] .- (num[:,3] .* num[j,4])) .+ (g1s[T,1,3] .+ g2s[T,1,3]) # gem w/ pac
+    end
+    @assert(all(combined .>= 0.0))
+    return combined
+end
```

Comment:
1. Subtracted the control cell # at time = 72hours.
2. for all 6 2-combinations for all 4 drugs, I wrote the bliss and added the control cell number back. But the assertion fails. The cell number goes negative

Reply:
Remember it needs to be a percent decrease. So, you divide by the control, then subtract off 1.0 (the control), then multiply by -1.0.

Reply:
![image](https://user-images.githubusercontent.com/34812750/76896111-7d785f80-684e-11ea-98ae-e1f95aba4404.png)
What I meant in the issue, was this:
The values for bliss combination are really large, I am not sure if they mean anything.

Reply:
Then something's wrong. You can't have more than a 100% decrease (so 1.0).

Reply:
Did I have to do this for the parameters, too?

Reply:
Why is the variable here 3D? Shouldn't it be drug A by drug B?

Reply:
I mean the percentage part.

Reply:
You're comparing this to your existing model using the parameters.

Reply:
Sorry for the late response-- I went to get my monitor and other stuff from the office.
It is not 3D, the third dimension just identifies this is the combination of which two drugs.